Makefile

```diff
@@ -1,7 +1,7 @@
 GO ?= go
 GOBIN ?= $$($(GO) env GOPATH)/bin
 GOLANGCI_LINT ?= $(GOBIN)/golangci-lint
-GOLANGCI_LINT_VERSION ?= v1.43.0
+GOLANGCI_LINT_VERSION ?= v1.46.0
```

Comment:
Could we have latest version for golangci lint (currently v1.49.0)?

Reply:
upgraded in a different branch, closing this one for now